doc/user-guides/multi-cluster.md

```diff
@@ -0,0 +1,470 @@
+
+```
+kubectl apply -f - <<EOF
+apiVersion: kuadrant.io/v1alpha1
+kind: DNSPolicy
+metadata:
+  name: loadbalanced
+  namespace: ${gatewayNS}
+spec:
+  routingStrategy: loadbalanced
+  loadBalancing:
+    geo: 
+      defaultGeo: US 
+    weighted:
+      defaultWeight: 120 
+  targetRef:
+    name: external
+    group: gateway.networking.k8s.io
+    kind: Gateway
+EOF
+```    
+Note: the DNSPolicy will leverage the ManagedZone we defined earlier based on the listener hosts defined in the gateway.
+
+Lets check our DNSPolicy has been accepted.
+```
+kubectl get dnspolicy loadbalanced -n ${gatewayNS} -o=jsonpath='{.status.conditions[?(@.type=="Accepted")].message}'
+```
+
+
+#TODO add section about viewing Gateway dashboards
+
+## Platform Engineer review
+
+So far we have setup an external gateway, secured it with TLS, Protected all endpoints with a default `DENY ALL` AuthPolicy added a restrictive default RateLimitPolicy and set up ManagedZone and a DNSPolicy to ensure traffic is brought to the gateway for the listener hosts defined. Our gateway is now ready to start accepting traffic.
+
+Once we create a HTTPRoute for our listeners, it will cause the DNSPolicy, Auth and RateLimitPolicy to be `Enforced`. So DNS records will populate auth and rate limiting will be configured and ready to protect requests to that endpoint. 
+
+We can test this by deploying a simple application and connecting it to our gateway.
+
+```sh
+kubectl apply -f https://raw.githubusercontent.com/Kuadrant/Kuadrant-operator/main/examples/toystore/toystore.yaml -n ${gatewayNS}
+```
+
+add a HTTPRoute
+
+```
+kubectl apply -f - <<EOF
+apiVersion: gateway.networking.k8s.io/v1
+kind: HTTPRoute
+metadata:
+  name: api
+  namespace: ${gatewayNS}
+spec:
+  parentRefs:
+  - name: external
+    namespace: ${gatewayNS}
+  hostnames:
+  - "toys.${rootDomain}"
+  rules:
+  - backendRefs:
+    - name: toystore
+      port: 80
+EOF
+```
+
+Ok lets check our gateway policies are enforced
+
+```
+kubectl get dnspolicy loadbalanced -n ${gatewayNS} -o=jsonpath='{.status.conditions[?(@.type=="Enforced")].message}'
+kubectl get authpolicy external -n ${gatewayNS} -o=jsonpath='{.status.conditions[?(@.type=="Enforced")].message}'
+kubectl get ratelimitpolicy external -n ${gatewayNS} -o=jsonpath='{.status.conditions[?(@.type=="Enforced")].message}'
+```
+note TLS policy is currently missing an enforced condition. https://github.com/Kuadrant/kuadrant-operator/issues/572. However looking at the gateway status we can see it is affected by 
+
+```
+kubectl get gateway -n ${gatewayNS} external -n demo -o=jsonpath='{.status.conditions[*].message}'
+```
+
+### Test connectivity and deny all auth 
+
+We are using curl to hit our endpoint. As we are using letsencrypt staging in this example we pass the `-k` flag.
+
+```
+curl -k -w "%{http_code}" https://$(kubectl get httproute api -n demo -o=jsonpath='{.spec.hostnames[0]}')
```

Comment:
$gatewayNS should be used instead of the `demo` namespace.

I am a bit surprised that `-k` switch needs to be used, I thought that thanks to using Let's Encrypt it is not needed. Without it I am getting `unable to get local issuer certificate`. Not sure if that's expected. Could it be due to using staging LE url rather that production one in `clusterIssuer` ?